### DISCUSSION TITLE
Installation Guide for a beginner???

### DISCUSSION THREAD

**Discussion by s3nox** [tphakala/birdnet-go#84] [General]
To qualify this, I am not an absolute beginner. I successfully have three Raspberry Pi projects up and running but your installation guide assumes a far more advanced level of knowledge than mine. My first project, MotionOS was easy. Volumio with a touch screen and a DAC fun. An ADSB receiver not at all difficult. I feel I ought to be able to install BirdNet Go but I need help getting started.

I was looking at BirdNet-Pi which seems to be no longer supported. It had some basic installation instructions [here](https://github.com/mcguirepr89/BirdNET-Pi/wiki/Installation-Guide). I am afraid I can't follow your installation guide.

Your installation guide talks of using "Docker" in both linux and windows which confuses me. I don't know what docker is, though Google tells me it is a container used for running apps?

Any chance of a few pointers to get me started? I don't mind doing my own research online but bear in mind the last time I did any programming was in 1982 using BASIC. My hope is to set something up in my garden to give me an accurate picture of the birds I have visiting. I will be experimenting with various different microphones at the less expensive end of the market, possibly even building my own from an off the shelf waterproof mic capsule. As an ex-broadcast engineer my interest is mainly in the hardware rather than software.

**Comment by farski** [comment]
Discussion https://github.com/tphakala/birdnet-go/discussions/81 is covering a very similar topic, so it may be best to continue this discussion there.

As an aside, don't expect any BirdNET setups to give a _completely_ "accurate picture" of the birds in your area on its own. It can help you get a sense, but there will be times that a flock of 200 birds comes through your backyard and none of them get picked up. Some birds are just very quiet. If you want to record all the birds you've got, you have to make various types of observations and combine the results.
